.gitignore

```diff
@@ -0,0 +1 @@
+testing.js
```

Comment:
Do you have a test script?
Perhaps it would be interesting to have a test script, right?